tests/index.js

```diff
@@ -257,6 +258,23 @@ describe("blocks :: init with defaults", async assert => {
       },
     })
 
+    assert({
+      given: "form encoded with custom keywords",
+      should: "validate input",
```

Comment:
I would be more descriptive here (prob applies for my other tests as well), something like:

```
should: "transform and validate data inside schema with added functionalities from ajv-keywords"
```